.github/workflows/e2e-android.yml

```diff
@@ -49,9 +49,12 @@ jobs:
 
           # Temporarily disable checking for EPIPE error and use yes to accept all licenses
           set +o pipefail
-          yes | sdkmanager "platform-tools" "platforms;android-${{ env.ANDROID_COMPILE_SDK_VERSION }}"
+          yes | sdkmanager --licenses
```

Comment:
oh so this will work for any other subsequent runs?
even if we ask to install something new?
I thought it was per component.

Reply:
I believe yes, it will work. Note I changed the subsequent platform installation to not have the `yes |` prefix.
And Google [recommends](https://developer.android.com/tools/sdkmanager#accept-licenses) it especially for the CI:
> for a CI server or other headless Linux device without a GUI installed, do the following from the command-line:
sdkmanager --licenses